earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
+
+TMPFILE=/tmp/FusionTutorial-Full.tar.gz
+
 if [ ! -d Imagery ]; then
```

Comment:
its still not clear to me why checking for Imagery and not  the tar file ?
Also - what part of the code would actually copy the /download_tutorial.sh  into the tmp_install ?